log-config-prod.yaml

```diff
@@ -1,18 +1,9 @@
 rotation:
     logLimit: 5242880 # 5MB
     keepFiles: 20
-severity: Info
+severity: Debug
```

Comment:
Are you sure? 🤔

Reply:
Read vatnik.

Reply:
Previously we had `Debug` for `node` (where almost all messages go) and `Info` for the rest (where ε messages go). Now we have `Debug` for all. It's quite minor.